pipelines/pingcap/tidb/release-6.2/pod-ghpr_unit_test.yaml

```diff
@@ -0,0 +1,60 @@
+apiVersion: v1
+kind: Pod
+spec:
+  securityContext:
+    fsGroup: 1000
+  containers:
+    - name: golang
+      image: "hub.pingcap.net/wangweizhen/tidb_image:20220823"
+      tty: true
+      resources:
+        requests:
+          memory: 8Gi
+          cpu: "4"
+        limits:
+          memory: 32Gi
+          cpu: "8"
+      command: [/bin/sh, -c]
+      args: [cat]
+      env:
+        - name: GOPATH
+          value: /share/.go
+        - name: GOCACHE
+          value: /share/.cache/go-build
+      volumeMounts:
+        - name: bazel-out
+          mountPath: /home/jenkins/.tidb
+        - name: bazel
+          mountPath: /data/
+          readOnly: true
+        - name: gocache
+          mountPath: /share/.cache/go-build
+        - name: gopathcache
+          mountPath: /share/.go
+    - name: net-tool
```

Comment:
What is the purpose of this container net-tool?

Reply:
dig github reachable, current maybe may network problem(dns or github reachable, if any command need add)
image size is 23MB